packages/shared/src/hooks/useAttachments.ts

```diff
@@ -50,20 +47,15 @@ export const useAttachments = (appBridge: AppBridgeBlock, blockSettingsKey: stri
         for (const asset of assets) {
             newAssetIds.push(asset.id);
         }
```

Comment:
could be something like that:

```suggestion
        const newAssetIds = assets.map((asset) => asset.id);
```